pkg/client/clientset/versioned/typed/cr/v1alpha1/fake/fake_actionset.go

```diff
@@ -36,9 +36,9 @@ type FakeActionSets struct {
 	ns   string
 }
 
-var actionsetsResource = schema.GroupVersionResource{Group: "cr.kanister.io", Version: "v1alpha1", Resource: "actionsets"}
+var actionsetsResource = schema.GroupVersionResource{Group: "cr", Version: "v1alpha1", Resource: "actionsets"}
```

Comment:
We have some issues with the directory layout. Thats why we don't commit changes in `fake` packages as it sets the wrong group name. Can you please run unit tests? We might have to revert this.

Reply:
unit tests failed in CI. will revert the changes in fake package